.github/dependabot.yml

```
@@ -0,0 +1,82 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

우리 일단 시작은 매일 체크하고 너무 자동 PR이 많이 올라오면 주기를 늘리는 게 어때요? 현재는 개발 일감이 많이 없어서 PR이 많이 올라오면 오히려 더 좋을 것 같아요 ㅋㅋ

```suggestion
      interval: "daily"
```